### PR TITLE
Added Player FAQ & info about how to link Discord account

### PR DESCRIPTION
**Pull request by Starystars67**
I also fixed some links in this.

Please can we get as many locales updated to support this PR before merging?

@lionkor @AlexITA1100 maybe I could ask you to assist in getting some of the translations done/checked? (IT,RU,DE)

### CODE REVIEW

**Comment by lionkor** [comment]
@Dash-kor

File: docs/en/support/player-faq.md
Comment (AlexITA1100) [inline docs/en/support/player-faq.md:1]: The following paragraphs should not be here as they're already in `game-faq.md`. This file should only answer questions about the player (sign up/linking Discord/Early Access benefits), not the compatibility of the game nor its performance.
- My game keeps freezing when other players edit or spawn vehicles
- How do I install BeamMP?
- Does BeamMP works on a cracked version of BeamNG.drive?
- Does BeamMP work on Linux?
- Why is the launcher flagged by my antivirus or by Windows Defender?
- I have poor performance / low FPS, what should I do?

**Review by AlexITA1100 — CHANGES_REQUESTED** [review]

File: docs/en/support/player-faq.md
Comment (Starystars67) [reply docs/en/support/player-faq.md:1]: Spot on, yep agreed.

File: docs/en/support/player-faq.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## I have poor performance / low FPS, what should I do?
+
+We are working hard to make performance as good as possible. Sadly, there isn't a lot you can do about it, if you have already lowered your graphic settings and your performance is still poor, consider playing on a server with less players. The game is mainly CPU bound when you play with lots of people, so older CPUs (even quadcores) will suffer with more than a handful of people.
+
+## How do I get early access?
+
+Early access (including the purple nametag and other benefits) can be obtained by either boosting the discord server or by helping to support us financially on [Patreon](https://patreon.com/BeamMP)
+
+## I have more questions?
```
[the remark this answers is not in the document]
Comment (Starystars67) [reply docs/en/support/player-faq.md:20]: Done something along these lines.

File: docs/en/support/player-faq.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Please ensure you do the following to automatically receive your perks:
+
+1. Link your Discord account on Patreon to receive the Roles and Access in the Discord server.
+2. Please ensure you use the same email address on Patreon as you do for your BeamMP account on the [Forum](https://forum.beammp.com/).
+
+Please be patient, the system checks approximately every 4 hours. If you have not received your perks after 12 hours and have completed the above steps please contact BeamMP support.
+
+## I have more questions?
+
+If your question or issue relates to the Game or playing please refer to the [Game FAQs](support/game-faq.md).
```
Comment (AlexITA1100) [inline docs/en/support/player-faq.md:22]: This leads to 404, should be only `(game-faq.md)`, sorry about that.

File: docs/en/support/player-faq.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+1. Link your Discord account on Patreon to receive the Roles and Access in the Discord server.
+2. Please ensure you use the same email address on Patreon as you do for your BeamMP account on the [Forum](https://forum.beammp.com/).
+
+Please be patient, the system checks approximately every 4 hours. If you have not received your perks after 12 hours and have completed the above steps please contact BeamMP support.
+
+## I have more questions?
+
+If your question or issue relates to the Game or playing please refer to the [Game FAQs](support/game-faq.md). 
+If your question or issue relates to running a Server please refer to the [Server FAQs](support/server-faq.md).
```
Comment (AlexITA1100) [inline docs/en/support/player-faq.md:23]: This leads to 404, should be only `(server-faq.md)`, sorry about that.

**Review by AlexITA1100 — APPROVED** [review]

**Comment by Starystars67** [comment]
Lets just wait on Russian translation for this too and then we can merge, thank everyone

**Pull request merged**